### PR TITLE
refactor: lab-09-5-sigmoid_back_prop.py

### PR DESCRIPTION
**Pull request by kkweon**
# refactor: reference, comment, loss,

# 1. Filename has changed
  - Reason: it's actually a sigmoid backpropagation (#51)
  - From `lab-09-5-softmax_back_prop.py`
  - To: `lab-09-5-sigmoid_back_prop.py`

# 2. Description of file was added
  - Reason: to give a clear idea of back propagation steps by explicitly defining a loss function and so on(#25)
  - Add [Prof. Sung KIM's slides](https://docs.google.com/presentation/d/1_ZmtfEjLmhbuM_PqbDYMXXLAqeWN0HwuhcSKnUQZ6MM/edit#slide=id.p6) and others
  - Define a loss function and a network architecture

# 3. Loss function change
  - Reason: In [Prof. Sung KIM's slides](https://docs.google.com/presentation/d/1_ZmtfEjLmhbuM_PqbDYMXXLAqeWN0HwuhcSKnUQZ6MM/edit#slide=id.p6), `reduce_sum` was used instead of `reduce_mean`
  - In this file, it's also modified to use `reduced_sum` to sync the slides
  - Although there is no difference in terms of convergence, a lower learning rate is required because  much larger numbers are now back propagated

# 4. Refactor X, y variable name
  - Scikit Learn's way to differentiate matrices and vectors
  - When we say 'X", the naming convention is to use uppercase because it's a multidimensional matrix
  - When we say "y", the naming convention is to use lowercase because it's a single column vector
  - With the exception of onehot vectors, we consider it as a single column vector hence lowercase

### CODE REVIEW

File: lab-09-5-sigmoid_back_prop.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+We will use the Sigmoid Cross Entropy loss function.
+This is equivalent to tf.nn.sigmoid_softmax_with_logits(logits, labels)
+
+[References]
+
+1) Tensorflow Document (tf.nn.sigmoid_softmax_with_logits)
+    https://www.tensorflow.org/api_docs/python/tf/nn/sigmoid_cross_entropy_with_logits
+
+2) Neural Net Backprop in one slide! by Prof. Sung KIM
```
Comment (hunkim) [inline lab-09-5-sigmoid_back_prop.py]: I think we can cut Prof. Just Sung Kim or Just "Neural Net Backprop in one slide!" :-)

**Comment by hunkim** [comment]
Wonderful! I just have one comment. :-)

File: lab-09-5-sigmoid_back_prop.py (hunk 1 above)
Comment (kkweon) [inline lab-09-5-sigmoid_back_prop.py]: Fixed!

**Comment by hunkim** [comment]
Lovely!
